### PR TITLE
Feature: Display quotes

### PR DESCRIPTION
**Pull request by Akorede3133**
In this PR, the following were affected:
* Fetched data inside of a new component(Quote)
* Showed a loading state
* Showed an error state
* Imported the new component into your main component (next to the Calculator)

### CODE REVIEW

File: src/components/Error.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
+const Error = ({ message }) => (
+  <div className=" p-2 text-white text-5xl">
+    Error:
+    {message?.message}
+  </div>
```
Comment (Gambit142) [inline src/components/Error.jsx]: - [ ] Kindly remove the `eslint-disable` statements used in this file. `Eslint` helps developers write clean, readable and industry-standard codes and using `eslint-disable` statements defeats this purpose. Kindly remove them. To solve the prop-types issue, you can read this [documentation](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html). 👍 😎

File: src/components/Quotes.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <p className="text-white text-sm italic">{quote}</p>
+      <p className=" self-end pt-2 italic text-slate-200 text-sm">
+        Author:
+        <span>{` ${author}`}</span>
+      </p>
+    </div>
+  );
+};
+
+export default Quotes;
```
Comment (Gambit142) [inline src/components/Quotes.jsx]: - [ ] Kindly remove the `eslint-disable` statements used in this file. `Eslint` helps developers write clean, readable and industry-standard codes and using `eslint-disable` statements defeats this purpose. Kindly remove them. To solve the prop-types issue, you can read this [documentation](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html). 👍 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Akorede3133 👋, 

Good job so far!

## Highlights
- Setup linters correctly ✔️ 
- Superb use of react components ✔️ 
- Setup the API correctly ✔️ 
- Renders the Quote successfully ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Akorede3133 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**